src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-example.mdx

```diff
@@ -1,10 +1,5 @@
 ---
-title: Instrument a Lambda function
-tags:
-  - Serverless function monitoring
-  - AWS Lambda monitoring
-  - Enable Lambda monitoring
-  - Instrumentation
+title: "Enable Lambda monitoring step 2: Instrument a Lambda function"
```

Comment:
@zuluecho9, could we shorten this title to this: "Step 2: Instrument a Lambda function"? I'm guessing that users can get the context from the title and from the left navigation.